### PR TITLE
Update WDL tasks to comply with dxCompiler restrictions

### PR DESCRIPTION
**Pull request by adthrasher**
To support a user, we want to be able to use rnaseq-standard-fastq with dxCompiler. dxCompiler sets stricter requirements than miniwdl or cromwell. This PR addresses those requirements.

### CODE REVIEW

File: workflows/chipseq/chipseq-bwa-db-build.wdl [hunk 1]
```diff
@@ -48,7 +48,7 @@ workflow bwa_db_build {
     }
 
     call util.download as reference_download { input: url=reference_fa_url, outfilename=reference_fa_name }
-    call bwa.build_db as build_bwa_db {
+    call bwa.build_bwa_db as build_bwa_db {
```
Comment (a-frantz) [inline workflows/chipseq/chipseq-bwa-db-build.wdl:51]:
```suggestion
    call bwa.build_bwa_db {
```

File: workflows/general/bwa-db-build.wdl [hunk 2]
```diff
@@ -48,7 +48,7 @@ workflow bwa_db_build {
     }
 
     call util.download as reference_download { input: url=reference_fa_url, outfilename=reference_fa_name }
-    call bwa.build_db {
+    call bwa.build_bwa_db as build_db {
```
Comment (a-frantz) [inline workflows/general/bwa-db-build.wdl:51]:
```suggestion
    call bwa.build_bwa_db {
```
I'd rather use the more verbose name than create a more vague alias

File: workflows/rnaseq/rnaseq-star-db-build.wdl [hunk 3]
```diff
@@ -56,7 +56,7 @@ workflow rnaseq_star_db_build {
 
     call util.download as reference_download { input: url=reference_fa_url, outfilename=reference_fa_name }
     call util.download as gtf_download { input: url=gtf_url, outfilename=gtf_name }
-    call star.build_db as star_db_build {
+    call star.build_star_db as star_db_build {
```
Comment (a-frantz) [inline workflows/rnaseq/rnaseq-star-db-build.wdl:59]:
```suggestion
    call star.build_star_db {
```
Using this name will look cleaner.

**Review by a-frantz — CHANGES_REQUESTED** [review]
I was worried this PR would have ugly changes needed for compliance, but all these changes can stand as their own as making cleaner code. Pleasantly surprised

**Comment by adthrasher** [comment]
> I was worried this PR would have ugly changes needed for compliance, but all these changes can stand as their own as making cleaner code. Pleasantly surprised

Personally, I dislike the namespacing related changes `bwa.build_db` -> `bwa.build_bwa_db` feels like adding redundant information. Otherwise I agree and I was pleasantly surprised how little modification was required.

**Review by a-frantz — APPROVED** [review]

**Pull request merged**